docs/getting-started/installation/mac.md

```diff
@@ -131,16 +93,48 @@ To force the platform to use `ruby`, you can either:
 	 test and development, use a file named .env.local, or modify .env.test.local
 	 and .env.development.local.
 
-5. Run `bin/setup`
+4. Do final application setup (including the installation of requisite Ruby gems and NodeJS modules) with `./bin/setup`.
+
+## Start Up Your Forem!
+
+```sh
+./bin/startup
+```
+
+Presuming no errors here, your Forem should be accessible at `http://localhost:3000`.
+
+To run unit tests, first, prepare the database:
+
+```sh
+./bin/rails db:test:prepare RAILS_ENV=test
+```
+
+Then, try running any test, for example:
+
+```sh
+./bin/rspec spec/lib/acts_as_taggable_on
+```
+
+To shut down your Forem stack when you're done tinkering, shut services down in the inverse order of how we started them:
```

Comment:
Should we include an instruction that they can have these services always on?
```
brew services start postgresql
brew services start redis-server
```

Reply:
No, because these services aren't installed with Homebrew, and the system services that Homebrew installs point to the binaries that Homebrew installs. Without those binaries installed, there's no service definitions at the system-level, and the CLI is the only way to start these versioned deps. (Further, even if the user *does* have Postgres installed system-wide with Homebrew, the system-wide service would be starting that (wrong) system-wide version, and not the version RTX just installed)

Last thought, is that this makes the devxp similar between Linux and MacOS. I looked into pulling something like s6 or even supervisord to manage these daemons, but ultimately decided the lowest common denominator - the daemonization each tool already provides - was the better option for having consistent instructions, and fewer moving pieces to go wrong.